fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/CreateMTMVInfo.java

```diff
@@ -195,6 +197,16 @@ public void analyze(ConnectContext ctx) throws Exception {
         rewriteQuerySql(ctx);
     }
 
+    /**validate column name*/
+    public void validateColumns(List<ColumnDefinition> columns) throws UserException {
+        Set<String> colSets = Sets.newTreeSet(String.CASE_INSENSITIVE_ORDER);
+        for (ColumnDefinition col : columns) {
+            if (!colSets.add(col.getName())) {
```

Comment:
do we check invalid character in column name?

Reply:
Yeah,already have the logic to check invalid caracter in column name, and move the check invalid caracter logic to method `validateColumns`